stl/inc/format

```diff
@@ -1196,12 +1204,16 @@ _OutputIt _Write(_OutputIt _Out, _Arithmetic _Value) {
     array<char, _Format_min_buffer_length> _Buffer;
     const auto [_End, _Ec] = _STD to_chars(_Buffer.data(), _Buffer.data() + _Buffer.size(), _Value);
     _STL_ASSERT(_Ec == errc{}, "to_chars failed");
-    return _STD copy(_Buffer.data(), _End, _Out);
+    return _STD _Copy_and_widen<_CharT>(_Buffer.data(), _End, _Out);

```

Comment:
Now that `_Copy_and_widen` is `_Ugly`, it doesn't need to be `_STD` qualified. Ditto below.